syntax = "proto2";

package g2p;

enum WorkerStatus {
    IDLE = 0;
    WORKING = 1;
    ERROR = 2;
    DONE = 3;
    PENDING = 4;
    DISCONNECTED = 5;
    SHUTTING_DOWN = 6;
    RECONNECTING = 7;
}

